modules/wallet/encrypt_test.go

```diff
@@ -0,0 +1,272 @@
+package wallet
+
+import (
+	"bytes"
+	"crypto/rand"
+	"errors"
+	"path/filepath"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/consensus"
+	"github.com/NebulousLabs/Sia/modules/gateway"
+	"github.com/NebulousLabs/Sia/modules/miner"
+	"github.com/NebulousLabs/Sia/modules/transactionpool"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// For some of the encryption testing, a wallet that has not been encrypted,
+// unlocked, or been used to mine coins is necessary.
+func createBlankWallet(name string) (string, modules.ConsensusSet, modules.TransactionPool, modules.Miner, *Wallet, error) {
```

Comment:
The caller is going to have to defer `Close` calls on each of these. This is especially dangerous in tests where you don't assign all the variables (e.g. `_, _, _, _, w, err := createBlankWallet()`). I suggest defining a `blankWallet` object with a `Close` method. This would also spare you the ugliness of 6 return values, lol